config/crd/patches/webhook_in_managedfusionofferings.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patch enables conversion webhook for CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: managedfusionofferings.misf.ibm.com
+spec:
+  conversion:
+    strategy: Webhook
+    webhookClientConfig:
+      # this is "\n" used as a placeholder, otherwise it will be rejected by the apiserver for being blank,
+      # but we're going to set it later using the cert-manager (or potentially a patch if not using cert-manager)
+      caBundle: Cg==
+      service:
+        namespace: system
+        name: webhook-service
+        path: /convert
```

Comment:
Why are we enabling this?

Reply:
We haven't enabled this, it is generated along with API.
The file path is commented in kustomization.yaml
https://github.com/red-hat-storage/managed-fusion-agent/pull/8/files#diff-c025225277cbb9d6905457bdacd823dd511d3cd02f4da32362ed65f51572a23fR18

Reply:
Ok